src/main/java/core/basesyntax/dao/fileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class fileReaderImpl implements fileReader {
+
+    @Override
+    public String[] read(String nameOfFile) {
+       List<String> textInFile = null;
+        try(BufferedReader reader = new BufferedReader(new FileReader(nameOfFile))) {
```

Comment:
Using both `BufferedReader` and `Files.readAllLines()` is redundant. You can remove the `BufferedReader` as `Files.readAllLines()` already reads all lines from the file.